benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(STL_BENCHMARK_MSVC_RUNTIME_LIBRARY
+    MultiThreaded
+    CACHE STRING "The version of the standard library to use; see https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html for more information.")

```

Comment:
"version" seems confusing here (I expect that to refer to 17.2, 17.3, etc.). Would "flavor" be better?